apps/webview/src/app/[locale]/(home)/character-svg.tsx

```
@@ -0,0 +1,268 @@
+interface CharacterSVGProps {
+  svgContent?: string;
+}
+
+export const CharacterSVG = ({ svgContent }: CharacterSVGProps) => {
+  if (svgContent) {
+    return <div dangerouslySetInnerHTML={{ __html: svgContent }} />;
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)